source/frontend/public/tenants/tenant.json

```diff
@@ -14,7 +14,7 @@
   "layers": [],
   "propertiesUrl": "ogs-internal/ows?service=wfs&request=GetFeature&typeName=PIMS_PROPERTY_LOCATION_VW&outputformat=json&srsName=EPSG:4326&version=2.0.0&",
   "parcelMapFullyAttributed": {
-    "url": "https://openmaps.gov.bc.ca/geo/pub/WHSE_CADASTRE.PMBC_PARCEL_FABRIC_POLY_FA_SVW/ows",
+    "url": "https://openmaps.gov.bc.ca/geo/pub/WHSE_CADASTRE.PMBC_PARCEL_FABRIC_POLY_FA_SVW_TEST/ows",
     "name": "pub:WHSE_CADASTRE.PMBC_PARCEL_FABRIC_POLY_FA_SVW"
   }
```

Comment:
I believe this whole json file gets replaced in live environments via a config-map but shouldn't this placeholder json file have the new urls too?

Reply:
Any missing keys get filled in by the default object, so note required here.